subsys/bootloader/image/CMakeLists.txt

```diff
@@ -59,8 +59,15 @@ if (s1_addr)
   set(s1_arg --s1-addr ${s1_addr})
 endif()
 
+if (CONFIG_SOC_NRF9160)
+  set(provision_addr "0xff8108")
```

Comment:
This sure looks random, should we add a comment?

Reply:
Added intermediate variable and a comment. Tell me if more should be added.